resources/install.rb

```diff
@@ -0,0 +1,110 @@
+provides :pyenv_install
+# provides :pyenv_user_install
+# provides :pyenv_system_install
+unified_mode true
+
+property :prefix_type,
```

Comment:
I think we should pick a 'standard' property definition format as there's 3 types alone in this cookbook 😂. My vote would be for the one on line 17 here where the name and types are on the first line with the rest of the options on subsequent lines as it's probably the most widespread format across most cookbooks.

Reply:
Yeah I've been trying to just get this working rather than focus on style here 👍🏼